doc/api.yml

```diff
@@ -2304,6 +2388,20 @@ components:
         creation_datetime:
           type: string
           format: date-time
+    FeeType:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        total_amount:
+          type: integer
+        number_of_months:
+          type: integer
+        creation_datetime:
+          type : string
+          format : date-time
```

Comment:
separate into two entities, one CreateFeeType without the creation_datetime and inherit the properties of CreateFeeType for FeeType